safe-global/safe-wallet-web
# feat: new spending limit flow

## What it solves

Migrating the new/update spending limit flow  to the new transaction flow framework.

## How this PR fixes it

The new/update spending limit flow has been ported, as well as removing the old one.

## How to test it

Create a new/update a spending limit via the settings and observe a successful transaction.

## Screenshots

![new spending limit flow](https://github.com/safe-global/safe-wallet-web/assets/20442784/065770a1-84af-4ce2-bbe7-d76213ae106c)

## Checklist
